README.md

```diff
@@ -306,6 +306,24 @@ Functions that have `groupedData` as parameter, means that `groupedData` is a JS
   - `continent`: Continent name that came from the GeoJSON map file.
 - `longitudeKey:` _(string, default: 'longitude')_ Column name of the longitude (from the dataset).
 - `latitudeKey:` _(string, default: 'latitude')_ Column name of the latitude (from the dataset).
+- `defaultScaleTo:` _(object, default:null)_ Default zoom state for the map based on either Country code or Latitude/Longitude point.
+
+  - `defaultScaleTo.scaleType:` _(string, default:null)_ Type of a zoom/scale on the map. This only accepts two types: 'lat/lng' for a zoom based on latitude and longitude points or 'country' for a zoom based on specific country code(iso_a3 code).
+  - `defaultScaleTo.values` _(object, default:null)_ Option values for the zoom state on the map based on 'scaleType' property. If 'scaleType' is set to 'lat/lng' then its 'values' object should contain: 'latitude','longitude' and 'scale' properties. If 'scaleType' is set to 'country' then its 'values' object should contain: 'iso_a3' property.
+
+    _Example:_
+
+    ```js
+    defaultScaleTo: {
+      scaleType: 'lat/lng',
+      values: {
+        latitude: '0',
```

Comment:
why are the coordinates and values not typed as Float instead of string?

Reply:
Updated